python-frontend/typeshed_serializer/serializer/typeshed_serializer.py

```diff
@@ -129,14 +151,16 @@ def serialize_typeshed_stdlib_multiple_python_version():
 
 
 def save_merged_symbols(is_debug=False, is_third_parties=False):
+    dir_name = "third_party_protobuf" if is_third_parties else "stdlib_protobuf"
     merged_modules = symbols_merger.merge_multiple_python_versions(is_third_parties)
     for mod in merged_modules:
-        symbols.save_module(merged_modules[mod], is_debug=is_debug, debug_dir="output_merge", is_stdlib=not is_third_parties)
+        symbols.save_module(merged_modules[mod], dir_name, is_debug=is_debug, debug_dir="output_merge")
 
 
 def main():
-    save_merged_symbols()
-    save_merged_symbols(is_third_parties=True)
+    #save_merged_symbols()
```

Comment:
Unnecessary change?

Reply:
Sorry, my mistake when reordering the commit, somehow this changed gets reverted in the .protobuf bulk update instead of not existing at all...